src/util/dom.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @param {HTMLElement | Array | String} target
+ * @param {Document} document
+ */
+export const domArray = (target, document) => {
+  if (typeof target === 'string') return Array.from(document.querySelectorAll(target))
+  if (target instanceof HTMLElement) return [target]
+  if (target.jquery) return target.toArray()
+  if (Array.isArray(target)) return target
+  return []
```

Comment:
Maybe we should remove that default array fallback

Reply:
I think that’s ok, but would probably document it.

Reply:
I've made it more compatible and just replaced it with `Array.from(target)`.
That way it's compatible with `NodeList` and `jQuery` instances out of the box.